static/src/javascripts/projects/common/modules/experiments/tests/sticky-mpu.js

```diff
@@ -0,0 +1,44 @@
+define([
+    'common/utils/config'
+], function (
+    config
+) {
+    return function () {
+        this.id = 'StickyMpu';
+        this.start = '2015-03-09';
+        this.expiry = '2015-05-09';
+        this.author = 'Zofia Korcz';
+        this.description = 'Sticky mpu everywhere where possible instead of the standard RH mpu';
+        this.audience = 0.01;
+        this.audienceOffset = 0;
+        this.successMeasure = '';
+        this.audienceCriteria = '1% of US and UK edition';
+        this.dataLinkNames = '';
+        this.idealOutcome = '';
+
+        this.canRun = function () {
+            var isUkOrUs = false;
+
+            if (config.page.edition === 'UK' || config.page.edition === 'US') {
+                isUkOrUs = true;
+            }
+
+            return isUkOrUs;
```

Comment:
I will use Sam's solution, because it is more readable and it doesn't require to include another module.